Userland/Applications/Browser/History.cpp

```diff
@@ -13,21 +13,24 @@ void History::dump() const
     dbgln("Dump {} items(s)", m_items.size());
     int i = 0;
     for (auto& item : m_items) {
-        dbgln("[{}] {} {}", i, item, m_current == i ? '*' : ' ');
+        dbgln("[{}] {} '{}' {}", i, item.url, item.title, m_current == i ? '*' : ' ');
         ++i;
     }
 }
 
-void History::push(const URL& url)
+void History::push(const URL& url, String title)
```

Comment:
You have resolved the conversation but not updated the code :)

Reply:
My bad, fixing it right away. Thanks for you patience!